nixos/modules/services/networking/kea.nix

```diff
@@ -45,6 +51,17 @@ in
             '';
           };
 
+          configFile = mkOption {
+            type = nullOr path;
+            default = null;
+            description = ''
+              Kea Control Agent configuration as a path, see <link xlink:href="https://kea.readthedocs.io/en/kea-${package.version}/arm/agent.html"/>.
```

Comment:
I like having these links in the description, but it means we need to disable the manual sandboxing.

```
  meta.buildDocsInSandbox = false;
```

Reply:
Wait. That's already disabled, why are the manual builds failing then.

Reply:
Ah the option was incorrectly referred to in the linkend. Just pushed a fix for it, lets see if it builds now.